### PR TITLE
Add external link mapping to table of contents spec

### PR DESCRIPTION
**Pull request by arcadiafalcone**
## Why was this change made?
Partially addresses #1936 

## How was this change tested?
n/a

## Which documentation and/or configurations were updated?
n/a

### CODE REVIEW

File: spec/services/cocina/mapping/descriptive/mods/table_of_contents_spec.rb [hunk 1]
```diff
@@ -110,6 +110,27 @@
     end
   end
 
+  describe 'Link to external value only' do
+    it_behaves_like 'MODS cocina mapping' do
+      let(:mods) do
+        <<~XML
+          <tableOfContents xlink:href="http://contents.org/contents" />
+        XML
+
```
Comment (justinlittman) [inline spec/services/cocina/mapping/descriptive/mods/table_of_contents_spec.rb:119]: Missing an `end` here.

**Review by justinlittman — CHANGES_REQUESTED** [review]
Does this work or should it be an `xit`?

File: spec/services/cocina/mapping/descriptive/mods/table_of_contents_spec.rb (hunk 1 above)
Comment (arcadiafalcone) [reply spec/services/cocina/mapping/descriptive/mods/table_of_contents_spec.rb:120]: Fixed.

**Review by justinlittman — APPROVED** [review]

**Comment by justinlittman** [comment]
#2027 is for implementation.

**Pull request merged**